ndvi.json

```diff
@@ -52,7 +52,7 @@
         }
     ],
     "returns": {
-        "description": "A raster data cube containing the computed NDVI values.",
+        "description": "A raster data cube containing the computed NDVI values. Depending on the value passed to `target_band` the data cube has changed differently:\n\n* `target_band` is `null`: The data cube is missing the dimension of type `bands`, the number of dimensions decreased by one. The dimension properties (name, type, labels, reference system and resolution) for all other dimensions have not changed.\n* `target_band` is a string: The data cube has the same dimensions. The dimension properties have not changed, but  the number of dimension labels for the dimension of type `bands` has increased by one. The additional label has the name specified in `target_band`.",
```

Comment:
as noted above (and depending on what is decided): I think the tense of the verbs can be tweaked a bit here: now it's a bit a mix: `is missing`, `decreased`, `have not changed`, `has`, `has increased`

Reply:
Thanks, indeed I'm not 100% sure about the tenses. Could also be future "a dimension will be missing". Native speakers here?

Reply:
I'd stick to present tense. 
My go would be:

"description": "A raster data cube containing the computed NDVI values. 
The data cube results differently depending on the value passed to `target_band`:
\n\n* `target_band` is `null`: The data cube does not contain the dimension of type `bands` anymore, the number of dimensions decreases by one. The dimension properties (name, type, labels, reference system and resolution) for all other dimensions remain unchanged.
\n* `target_band` is a string: The data cube keeps the same dimensions. The dimension properties remain unchanged, but the number of dimension labels for the dimension of type `bands` increases by one. The additional label is named as specified in `target_band`."

Reply:
@przell 

> The data cube results differently depending on the value passed to target_band:" sounds incorrect.

Should it be: 

> The data cube results differ depending on the value passed to target_band: